otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
```

Comment:
what do you think about `ssi-dim-wallet-stub`? as it fits in the area of Self-sovereign identities (SSI).
https://github.com/eclipse-tractusx?q=ssi&type=all&language=&sort=

Reply:
Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27